resources/authgear/templates/en/web/verify_identity.html

```diff
@@ -48,49 +31,46 @@ <h1 class="title primary-txt">{{ template "verify-user-page-title--email" }}</h1
 <form class="vertical-form form-fields-container" method="post" novalidate>
 {{ $.CSRFField }}
 
-{{ if eq $.VerificationCodeChannel "sms" }}
-	<input
-		class="input text-input primary-txt"
-		type="text"
-		inputmode="numeric"
-		pattern="[0-9]*"
-		autocomplete="one-time-code"
-		autocapitalize="none"
-		name="x_code"
-		placeholder="{{ template "verify-user-placeholder" }}"
-		value="{{ $.VerificationCode }}"
-	>
-{{ end }}
-{{ if eq $.VerificationCodeChannel "email" }}
-	<input
-		class="input text-input primary-txt"
-		type="text"
-		autocomplete="one-time-code"
-		autocapitalize="characters"
-		name="x_code"
-		placeholder="{{ template "verify-user-placeholder" }}"
-		value="{{ $.VerificationCode }}"
-	>
-{{ end }}
-<button class="btn primary-btn align-self-flex-end" type="submit" name="x_action" value="{{ $.Action }}">{{ template "next-button-label" }}</button>
+<input
+	class="input text-input primary-txt"
+	type="text"
+	inputmode="numeric"
+	pattern="[0-9]*"
+	autocomplete="one-time-code"
+	autocapitalize="none"
+	name="x_code"
+	placeholder="{{ template "verify-user-placeholder" }}"
+	value="{{ $.VerificationCode }}"
+	{{ if eq $.Action "update_session_step" }}readonly{{ end }}
```

Comment:
Use `disabled` or update css makes readonly to make it looks different from normal input?

Reply:
We cannot use `disabled` because `disabled` input are NOT included in the form submission. I'll make `readonly` look the same as `disabled` instead.